docs/components/info-unit-groups-image-text.md

```diff
@@ -111,7 +114,7 @@ variations:
       * Headings: Any heading size may be used, recommend H2-6
     variation_jinja_code_snippet: >-
       *
-      [image-text-25-75-group.html](https://github.com/cfpb/cfgov-refresh/blob/e67d1ad321551c221c01eaa62589dfdd1177d1dc/cfgov/jinja2/v1/_includes/organisms/image-text-25-75-group.html)
+      [info-unit-group-2.html](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_includes/organisms/info-unit-group-2.html)
```

Comment:
The original templates linked to here are deprecated in favor of a single template that covers 25-75 and 50-50. Does it still make sense to link there from this page separately?

Reply:
The question aside, this doesn't appear to be working as expected, as it looks like this:

<img width="865" alt="Screen Shot 2019-09-26 at 10 46 57 AM" src="https://user-images.githubusercontent.com/704760/65698768-0a362380-e04b-11e9-952d-8574a05e9546.png">